test/imperative.js

```diff
@@ -192,3 +192,17 @@ metatests.testSync('testAsync must return a test', (test) => {
   test.ok(t);
   t.end();
 });
+
+metatests.test('Must catch unhandledExeptions', (test) => {
+  const t = new metatests.ImperativeTest('Throwing test', () => {
+    throw new Error();
+  }, { async: false });
+
+  setTimeout(() => {
+    test.assert(t.done, 'Must finish');
+    test.assertNot(t.success, 'Must be failed');
+    test.strictSame(t.results[0].type, 'unhandledExeption');
+    test.strictSame(t.results[0].success, false);
```

Comment:
Why not use `assertNot` here like above?

Reply:
I wanted to signify success === false, though I'm fine with either.